### PR TITLE
Fixed broken links

### PR DESCRIPTION
**Pull request by watucker**

### CODE REVIEW

File: fwtheme_django_ceda_serv/templates/fwtheme_django/layout.html [hunk 1]
```diff
@@ -43,10 +43,10 @@
         <ul class="navbar-nav">
           {% block navbar_options %}
           <li class="nav-item">
-            <a class="nav-link" href="http://www.ceda.ac.uk/blog/">About</a>
+            <a class="nav-link" href="http://www.ceda.ac.uk/news/">About</a>
```
Comment (gap736uk) [inline fwtheme_django_ceda_serv/templates/fwtheme_django/layout.html]: I think this one is incorrect... 

It should point to:
https://www.ceda.ac.uk/about/

for archive related pages we should check with Sam as there is https://archive.ceda.ac.uk/about/ , but that looks badly strucutred... I'll check with Sam and get back to this point.

Comment (gap736uk) [reply fwtheme_django_ceda_serv/templates/fwtheme_django/layout.html]: ok - https://archive.ceda.ac.uk/about/ is still to be used for archive sites.

**Review by gap736uk — CHANGES_REQUESTED** [review]

File: fwtheme_django_ceda_serv/templates/fwtheme_django/layout.html (hunk 1 above)
Comment (watucker) [reply fwtheme_django_ceda_serv/templates/fwtheme_django/layout.html]: Got it, thanks!

**Comment by watucker** [comment]
@gap736uk I added the new "about" link and swapped all the HTTP links with HTTPS ones. Pretty sure all the links are working but a second opinion would be welcome.

**Comment by gap736uk** [comment]
Looks good to me.

**Pull request merged**